### PR TITLE
fix: audio-to-audio examples

### PR DESCRIPTION
**Pull request by younver**
closes `Audio-to-audio` task of https://github.com/huggingface/hub-docs/issues/792

- fixed `akhaliq/speechbrain-speech-seperation` space since there was a build failure by duplicating the space since it's been a while since last contribution.
- changed `facebook/Hokkien_Translation` space since the space was not working anymore.
- added `Matthijs/cmu-arctic-xvectors` embeddings dataset

cc: @merveenoyan

### CODE REVIEW

File: tasks/src/audio-to-audio/data.ts [hunk 1]
```diff
@@ -44,11 +48,11 @@ const taskData: TaskDataCustom = {
 	spaces:       [
 		{
 			description: "An application that can separate speech.",
-			id:          "akhaliq/speechbrain-speech-seperation",
+			id:          "younver/speechbrain-speech-separation",
 		},
 		{
-			description: "An application that can translate from speech to speech between Hokkien and English.",
-			id:          "facebook/Hokkien_Translation",
+			description: "An application that built to style transfer to audio using Huggingface diffusers.",
```
Comment (merveenoyan) [inline tasks/src/audio-to-audio/data.ts:54]:
```suggestion
			description: "An application for audio style transfer.",
```

File: tasks/src/audio-to-audio/data.ts [hunk 2]
```diff
@@ -44,11 +48,11 @@ const taskData: TaskDataCustom = {
 	spaces:       [
 		{
 			description: "An application that can separate speech.",
```
Comment (merveenoyan) [inline tasks/src/audio-to-audio/data.ts:50]:
```suggestion
			description: "An application for speech separation.",
```

**Review by merveenoyan — APPROVED** [review]
Looks good to me!

**Comment by younver** [comment]
> Looks good to me!

Thanks for suggestions and the approval^^

**Pull request merged**